setup_dev.sh

```diff
@@ -31,7 +31,9 @@ sed -i "s@^  :password.*@  :password => '$mysql_congress_forms',@" config/databa
 
 HOME=/home/vagrant sudo -u vagrant /bin/bash <<EOF
 echo "Setting up RVM and Ruby..."
-curl -sSL https://get.rvm.io | bash -s stable
+gpg --keyserver hkp://keys.gnupg.net --recv-keys D39DC0E3
```

Comment:
Hey @Hainish sorry for the late response.  Thanks for the note - I will look into this more when I get the chance!